libcontainer/cgroups/fs/cpu.go

```
@@ -93,6 +115,11 @@ func (s *CpuGroup) Set(path string, r *configs.Resources) error {
 				return err
 			}
 		}
+		if burst != "" {
+			if err := cgroups.WriteFile(path, "cpu.cfs_burst_us", burst); err != nil {
+				return err
```

Should we ignore the error of `unix.ENOENT` here too?